docs/calamari/SDK.md

```diff
@@ -0,0 +1,292 @@
+
+// Sync to get latest data after transaction and check that it was successful.
+await privateWallet.walletSync();
+
+// The private balance of pDOL should decrease by 10 units.
+const newPrivateBalance = await privateWallet.getPrivateBalance(assetId);
+```
+
+### To Public
+
+This example converts 5 private pDOL to 5 public DOL.
+
+```javascript
+// DOL token
+const assetId = new BN("1");
+const amount = new BN("5000000000000000000");
+
+// Sync with most recent ledger state.
+await privateWallet.initialWalletSync();
+
+// Get private address
+const privateAddress = await privateWallet.getPrivateAddress();
+
+// Get private balance of DOL for given private address
+const privateBalance = await privateWallet.getPrivateBalance(assetId);
+
+// Convert 5 pDOL back to DOL
+await privateWallet.toPublicSend(assetId, amount, polkadotSigner, polkadotAddress);
+
+// Sync to get latest data after transaction and check that it was successful.
+await privateWallet.walletSync();
+
+// The private balance of pDOL should decrease by 5 units.
+const newPrivateBalance = await privateWallet.getPrivateBalance(assetId);
+```
+
+### Manta Utilities
+
+There also exists a `MantaUtilities` class with additional functions. Mainly for interacting publicly with the Manta ecosystem. This example demonstrates these functions. This example assumes the `MantaPrivateWallet` class has already been initialized, as well as `polkadotAddress` and `polkadotSigner`.
+
+```javascript
+import { MantaUtilities } from "manta.js";
+
+// Get signer version, signer must be running.
+const signerVersion = await MantaUtilities.getSignerVersion();
+
+// DOL token
+const assetId = new BN("1");
+
+// Get public balance of DOL for `polkadotAddress`.
+const oldPublicBalance = await MantaUtilities.getPublicBalance(privateWallet.api, assetId, polkadotAddress);
+
+// Public transfer of 5 DOL to `destinationAddress`.
+const destinationAddress = "dmyhNmYL13N7ZKcVYqBQhvrk5kSfrKZUmrjX9vAaM4846bWKR";
+const amount = new BN("5000000000000000000");
+await MantaUtilities.publicTransfer(privateWallet.api, assetId, amount, destinationAddress, polkadotAddress, polkadotSigner);
+
+// Public balance should now be 5 DOL less that `oldPublicBalance`.
+const newPublicBalance = await MantaUtilities.getPublicBalance(privateWallet.api, assetId, polkadotAddress);
+```
+
+### Sign and manually send transaction
+
+In some cases you may not want to send transaction to the ledger through manta.js, thus you can get sign result after manta-signer has signed the transaction and send the transaction yourself. This is done by using the `toPrivateBuild`, `privateTransferBuild`, `publicTransferBuild` functions.
+
+This example returns the signed transaction of `toPrivate` for 10 DOL.
+
+```javascript
+const assetId = new BN("1");
+const amount = new BN("10000000000000000000");
+
+const env = sdk.Environment.Development;
+const net = sdk.Network.Dolphin;
```

Comment:
Seems current doc is for calamari, `Dolphin` should be `Calamari`.

Reply:
Right, but Dolphin is the public testnet of Calamari, that's what the dolphin variant is for.

Reply:
Ok I just made the doc reference Calamari and Manta production.